<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3">
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="4" rowspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(48, 48, 222);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Принять</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дата:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" rowspan="2">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Логин волонтера:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" rowspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(48, 48, 222);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Добавить мероприятие в книжку волонтера:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Уровень:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" rowspan="2">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QRadioButton" name="radioButton">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Волонтер</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="5">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Изменить статус волонтера:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Название:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID волонтера:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(48, 48, 222);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateEdit" name="dateEdit">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Администратор</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
